projects/angular/src/layout/tabs/_tabs.clarity.scss

```
@@ -92,54 +81,38 @@ button.nav-link {
     box-shadow: none;
     flex-direction: column;
     align-items: stretch;
-    margin-right: variables.$clr_baselineRem_1;
+    margin-right: #{tokens.$cds-global-space-9};
```

Remove curly brackets.

```suggestion
    margin-right: tokens.$cds-global-space-9;
```